### ISSUE TITLE
[BUG]: Running channel `/` commands results in an error

### ISSUE THREAD

**Issue by jiriks74** [DynamicaBot/Dynamica-V2#391]
Using commands like `/bitrate`, `/limit`, etc. results in a following error:

```
Invalid Form Body
channel_id[NUMBER_TYPE_COERCE]: Value "undefined" is not snowflake.
```

<details>
<summary>Screenshot</summary>

![image](https://github.com/user-attachments/assets/f7b41251-abc4-409d-bb93-6afc549a85c3)

</summary>

**Comment by jiriks74** [comment]
Tested bot selfhosted and the official bot. The result is the same.

**Comment by sebasptsch** [comment]
> Tested bot selfhosted and the official bot. The result is the same.

Forgot to document but the not needs the guild members intent and the presence intent to work. Are those enabled?

**Comment by jiriks74** [comment]
> Forgot to document but the not needs the guild members intent and the presence intent to work. Are those enabled?

Yes, those are enabled

<details>
<summary>Screenshot</summary>

![image](https://github.com/user-attachments/assets/37bd5e10-bb1a-4d2c-8b60-d676e4d773df)

</details>

**Comment by sebasptsch** [comment]
I'll have a look at some point today (Australian time)

**Comment by jiriks74** [comment]
It seemds the `/` commands aren't configured properly. Normally I see a place for arguments but I don't see any for this bot

<details>
<summary>Other bots</summary

![image](https://github.com/user-attachments/assets/28453a40-f5d1-4fe0-9bf3-5c6e25e390cb)
![image](https://github.com/user-attachments/assets/5bb984fb-6af5-44c2-af25-e2697950add1)

</details>

<details>
<summary>Dynamica</summary>

![image](https://github.com/user-attachments/assets/e114f7cb-8c72-49a1-bc74-7c79e5f5d018)

</summary>

**Comment by jiriks74** [comment]
> I'll have a look at some point today (Australian time)

Thanks. I may not sleep today so I may be able to do some research/debugging for you (and potentially look into the code if I'll have the capacity). I'm CEST.

**Comment by jiriks74** [comment]
I'll try to take a look at the `/` command definitions but I don't do Javascript/TypeScript so no primises.

**Comment by sebasptsch** [comment]
All good, the primary libraries that are involved are probably DiscordJS/Necord

**Comment by jiriks74** [comment]
So far I see that you, *probably*, used an invalid option while setting up some options: https://github.com/DynamicaBot/Dynamica-V2/blob/d0157117307c1021b1f06505816e83ec464b4a84/src/dto/PrimaryInfoDto.ts#L8

**Comment by jiriks74** [comment]
Trying to figure out why this line has a `type` in it
https://github.com/DynamicaBot/Dynamica-V2/blob/d0157117307c1021b1f06505816e83ec464b4a84/src/commands/info.command.ts#L8
when the docs don't use it
[![docs](https://github.com/user-attachments/assets/08c1497c-5c34-42a4-a6b9-44ed75b24a79)](https://www.necord.org/interactions/slash-commands#options)

**Comment by sebasptsch** [comment]
I think I found it, I was importing some of my Dto's as only types which works in typescript but doesn't compile correctly with decorators. I've push a fix and triggered a release to try and fix it.

**Comment by sebasptsch** [comment]
Problem solved in #394

**Comment by jiriks74** [comment]
I can confirm that everything is working as it should. Thanks for the quick update.
